frontend/src/components/modal-compare/modal.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, Dispatch } from 'react';
+
+import fillCompare from '@assets/images/icon_balance.svg';
+
+import styles from './styles.module.scss';
+
+interface ModalProps {
+  carName: string;
+  carDescription: string;
+  isHidden: boolean;
+  setIsHidden: Dispatch<boolean>;
+}
+
+const Modal: FC<ModalProps> = ({
```

Comment:
This component is the `TOAST`, not the `MODAL`.
Toast is a non-intrusive, optional message for the user. But the modal totally intrusive visual component which requires interaction from the user and blocks out the rest of the page

Toast example:
![image](https://user-images.githubusercontent.com/68850090/186001501-ed7a980e-26b0-439b-a26d-9098a5d4ab71.png)

Modal: 
![image](https://user-images.githubusercontent.com/68850090/186001542-6f80f3de-f039-4f6e-906a-1eb07e572d54.png)

--- 

### So this component should be named `CompareModal`. Also, use the capital letter for the React components names